src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -159,6 +159,20 @@ private ClassifyTextResponse classify() throws IOException {
     return events;
   }
 
+  public Set<String> getGreetings() {
+    Set<String> greetings = new LinkedHashSet<String>();
+    String[] allGreetings = new String[] {"good morning", "congratulation", "welcome", "good evening", 
+                                          "good night", "happy holiday", "good afternoon", " hello", "hey"};
+    
+    for (int i = 0; i < allGreetings.length; i++) {
```

Comment:
nit: if you don't have to use the index for any operation nor modifying the array to iterate, but only dealing with elements, you can use a more idiomatic for loop using
for (type var : array)